builtin/providers/aws/resource_aws_db_subnet_group.go

```diff
@@ -166,8 +165,9 @@ func resourceAwsDbSubnetGroupUpdate(d *schema.ResourceData, meta interface{}) er
 		}
 
 		_, err := conn.ModifyDBSubnetGroup(&rds.ModifyDBSubnetGroupInput{
-			DBSubnetGroupName: aws.String(d.Id()),
-			SubnetIds:         sIds,
+			DBSubnetGroupName:        aws.String(d.Id()),
+			DBSubnetGroupDescription: aws.String(d.Get("description").(string)),
```

Comment:
Hi @ColinHebert 

This will only actually be called if there is a chance in subnet_ids

This update will need to be reworked. I will manually make those changes on the PR and test it.